src/Integration/LocalServices/IFileSystem.cs

```diff
@@ -0,0 +1,19 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IFileSystem.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace SonarLint.VisualStudio.Integration
+{
+    // Test wrapper over basic file system operations
+    internal interface IFileSystem
+    {
+        bool IsFileExist(string filePath);
```

Comment:
This method name reads very strangely. Simply `FileExists` would be better, and consistent with `DirectoryExists` and `System.IO.File.Exists`. The correct wording would be `DoesFileExist` but this also looks strange IMO.